app/src/main/java/com/example/habitsmasher/HabitEventList.java

```diff
@@ -133,16 +143,13 @@ public void editHabitEventLocally(String newComment, Date newDate, int pos) {
 
     /**
      * This method is responsible for editing a habit in the database
-     * @param editedHabitEvent
-     * @param username (String) The username of the current user
-     * @param newComment (String) The edited comment
-     * @param newDate (Date) The edited date
-     * @param toEditId (String) The ID of the habit event to edit
+     * @param editedHabitEvent (HabitEvent) The edited event
+     * @param userId (String) The username of the current user
      * @param userId (String) The id of the current user
      * @param parentHabit (Habit) The current habit
```

Comment:
The new parameter is missing from the javadoc.

- [ ] add new param to method header